unicoremc/utils.py

```diff
@@ -122,3 +126,47 @@ def get_default_docker_cmd(application_type, country):
             docker_cmd = constants.MARATHON_CMD % {
                 'config_path': ini}
     return docker_cmd
+
+
+def get_repos(refresh=False):
+    """
+    Fetches and returns a list of public repos from Github. Caches
+    the result.
+    """
+    if not refresh:
+        return cache.get('repos')
+    url = urljoin(
+        settings.GITHUB_API, 'repos?type=public&per_page=100&page=%s')
+    pageNum = 1
+    repos = []
+    while True:
+        response = requests.get(url % pageNum)
```

Comment:
@Rizziepit can you add the auth token here too please. There's a limit of 60 requests for unauthenticated requests